### ISSUE TITLE
In mozilla applications, nvda fails to speak, or speaks incorrectly the text backspaced

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3650]
**Reported by driemer.riemer@... on 2013-11-17 10:01**
In some edit fields in mozilla applications, Especially in the composition window of Mozilla thunderbird, nvda fails to report the text being deleted with backspace or ctrl+backspace. 
## how to reproduce

If you are lucky enough to experience this problem you likely will see it occur randomly so it may be a ******\* to debug.

**Comment by leonardder** [comment]
I'm not sure whether I've seen this, but at least I haven't considered that major to file an issue for it, and I'm using Thunderbird on a daily basis.

**Comment by jcsteh** [comment]
Where this does occur, it should be addressed by #6424. Closing as duplicate.

**Comment by zahra21** [comment]
hello.
i downloaded nvda 2017.3 two days ago just for testing and reverted to 2017.2
after my test, (i should mention that i am not familiar with thunderbird, my test was in firefox).
i realized that when i write some lines in edit boxes (specially in gmail and wordpress websites) when i navigate in my written documents, sometimes nvda report incorrect characters, words and lines, (i see mixture of content specially when i use persian and english both in my document or only i use persian content).
the problem for using backspace, is resolved for me many months ago, (i dont remember exactly in which version of firefox).
thanks extremely for your great screen reader, i always sincerely pray for you and request divine infinite mercy for you.

**Comment by zahra21** [comment]
hello.
i downloaded nvda 2017.3 just for testing and hears my observations.
i should mention that i am not familiar with thunderbird, (my test was in firefox only).
after my test, i realized that when i write some lines in edit boxes in all edit boxes, (i tested specially in gmail and wordpress websites) when i navigate in my written documents, sometimes nvda report incorrect characters, words and lines, (i see mixture of content specially when i use persian and english both in my document or only when i use persian content).
the problem for using backspace, is resolved for me many months ago, (i dont remember exactly in which version of firefox).
so, for this reason, i write my comment, messages etc in notepad plus plus, copy the content and peist in the edit boxes in mozilla firefox.
i returned to nvda 2017.2 (which is the best and my favorite version) that i decided to keep and use it forever by the Grace of God!
thanks extremely for your great screen reader, i always sincerely pray for you and request divine infinite mercy for you.
